Frameworks/UIKit/UIView.mm

```diff
@@ -627,24 +434,19 @@ - (UITouchPhase)_processPointerEvent:(WUXIPointerRoutedEventArgs*)pointerEventAr
             /////////////////////////////////////////////////////////////////////////////
             // TODO: This block will be removed once we move to WinRT gesture recognizers
             //  If all fingers come off the screen, reset all gestures
-            g_resetAllTrackingGestures = TRUE;
+            BOOL shouldResetTrackingGesturesList = YES;
             for (auto& touchPoint : s_touchPoints) {
                 if (touchPoint.touch && touchPoint.touch->_view) {
-                    g_resetAllTrackingGestures = FALSE;
+                    shouldResetTrackingGesturesList = NO;
                 }
             }
 
-            if (g_resetAllTrackingGestures) {
-                for (UIGestureRecognizer* curgesture in g_currentlyTrackingGesturesList) {
-                    [curgesture reset];
-                }
-
-                [g_currentlyTrackingGesturesList removeAllObjects];
+            if (shouldResetTrackingGesturesList) {
+                [[_UIGestureCoordinator singleton] resetTrackingGestureList];
             }
             // TODO: This block will be removed once we move to WinRT gesture recognizers
             /////////////////////////////////////////////////////////////////////////////
-
-            break;
+        } break;
```

Comment:
i think so.

---
In reply to: [100621728](https://github.com/Microsoft/WinObjC/pull/1909#discussion_r100621728) [](ancestors = 100621728)